src/memory/memory.ts

```diff
@@ -24,24 +23,32 @@ export class Memory {
 
   setData(address: Relocatable, data: MaybeRelocatable[]): void {
```

Comment:
it looks like we alternate with `data` and `value`, if we go with `value` or `segmentValue`, then let's use `value` and `values` everywhere

Reply:
`value` & `values` make more sense imo, but we'll still have an occurence of `data` in the project, as it is a field name of Cairo Zero compiled json.

No big deal though, as it can be limited to the (Cairo Zero) Program type and we use `value`/`values` everywhere else 🤔